### PR TITLE
No results view

### PR DESCRIPTION
**Pull request by micahtaylor**

### CODE REVIEW

File: ui/components/pagers.jsx [hunk 1]
```diff
@@ -10,6 +10,9 @@ export default function Pagers({ count }, { router }) {
 
   if (count === 0) {
     pageInt = 0;
+    return (
+      <div className="alert p1 m1 border">No requirements match your search, try removing some filters to see more results.</div>
```
Comment (micahtaylor) [inline ui/components/pagers.jsx]: @theresaanna I could use your expertise on how to handle this better.

Comment (micahtaylor) [inline ui/components/pagers.jsx]: I think I figured it out.

**Comment by cmc333333** [comment]
So we're on the same page, this is not going to be merged in favor of #417, yes?

**Comment by micahtaylor** [comment]
Yes!

**Pull request closed without merging**